### PR TITLE
Add notranslate class to addresses

### PR DESCRIPTION
**Pull request by gkovats**
Tied to Issue:
https://github.com/code4recovery/12-step-meeting-list/issues/1092

First PR, could be missing some steps, glad to adjust and fix. The issue was Google Translate goes nuts and translates native language addresses to the target language, but this obfuscates addresses. GPS / map tools drive off native language addresses.

Wrapping display of addresses with "notranslate" to deny Google Translate from translating native language addresses. This comes from the Issue, assuming this is sufficient - hard to test locally as Google Translate requires a public site.

### CODE REVIEW

File: src/components/Table.tsx [hunk 1]
```diff
@@ -82,12 +84,13 @@ export default function Table({
           type: 'inactive',
         });
       }
+      console.info(attendance);
```
Comment (joshreisner) [inline src/components/Table.tsx:87]: did you mean to leave this here?

**Review by joshreisner — CHANGES_REQUESTED** [review]
looks awesome, thanks! mind taking the `console.info` out though?

File: src/components/Table.tsx [hunk 2]
```diff
@@ -82,12 +84,13 @@ export default function Table({
...
       return (
         <div css={tableChicletsCss}>
-          {attendance.map(({ icon, text, type }, index) => (
+          {attendance.map(({ icon, text, type, noTranslate }, index) => (
             <span css={tableChicletCss(type)} key={index}>
               <Icon icon={icon} size={18} />
-              {text && <span>{text}</span>}
+              {text && <span className={(true === noTranslate) ? 'notranslate' : ''}>{text}</span>}
```
Comment (joshreisner) [inline src/components/Table.tsx:93]: minor nit: i might write this as `className={noTranslate ? 'notranslate' : undefined}` so react doesn't render the empty property

File: src/components/Table.tsx (hunk 1 above)
Comment (gkovats) [reply src/components/Table.tsx]: Nope, just sloppy. Thanks for the catch.

File: src/components/Table.tsx (hunk 2 above)
Comment (gkovats) [reply src/components/Table.tsx]: Didn't realize undefined stripped the property, thanks - updated.

**Comment by gkovats** [comment]
@joshreisner updated the two things. Not sure if you want commits squashed in PR, some repos ask for that. Let me know if anything else is outstanding.

**Review by joshreisner — APPROVED** [review]
🎉

**Comment by joshreisner** [comment]
thanks @gkovats ! i have to merge it because the tests aren't passing - we haven't refactored those from the last pr yet

i'll 'squash & merge' - no need to squash PRs before the merge though

**Pull request merged**